telegram/ext/jobqueue.py

```diff
@@ -126,7 +156,9 @@ def tick(self):
                 self.logger.debug('Running job %s', job.name)
```

Comment:
I'm using a separate `get_days` function here, because if I put it in the class, it won't be filtered out by the list comprehension, and I won't be able to easily get the values from the attributes defined in the class.